docs/SystemDesign/1-TsFile/2-Format.md

```diff
@@ -143,120 +143,126 @@ PageHeader Structure
 
```

Comment:
In the table of ChunkHeader, we should remove the row of Max Tombstone Time.
In the table of PageHeader, we need to change the order of last two rows.

Reply:
Fixed